res/layout/activity_registration_2nd_device_qr.xml

```diff
@@ -86,4 +86,15 @@
         android:layout_width="match_parent"
         android:layout_height="0dp"/>
 
+    <TextView
+        android:id="@+id/bottom_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
this is not centered horizontally

Reply:
i know - the rest of the text is not centered horizontally either.

i tried both, i think both variants are not bad and are reasonable